syntax = "proto3";

message MxPca9539rStore {
  repeated bool state = 1;
}
